### ISSUE TITLE
DOC: Mismatch between docs and help for parameter `-sprrad` / `-radius`

### ISSUE THREAD

**Issue by corneliusroemer** [iqtree/iqtree2#65]
In the online command reference (http://www.iqtree.org/doc/Command-Reference), you say there's a parameter `-sprrad` with the following description:
-sprrad | Specify SPR radius for the initial parsimony tree search. DEFAULT: 6
-- | --

Yet when I try this out with IQtree 2.2.0, I get an error that this parameter doesn't exist.
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/25161793/158085628-5b73c30e-4b6a-4182-9b4e-e94d7ea9abd1.png">

Using `iqtree -h` I noticed that this parameter seems to have been renamed to `-radius`:
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/25161793/158085642-5130b0f9-a04a-4e7c-a349-61d82da5be07.png">

You may want to update docs on this.

In which  version did the change happen? Why do you not just accept both at least for the time being, old `-sprrad` and new `-radius`?

**Comment by corneliusroemer** [comment]
Another docs/help mismatch seems to be for collapsing near-zero branches.

In the online docs, it states that the parameter is `-czb` but in the help it's called `--polytomy`. Which on is right?

Lastly, the help seems to specify double minus for most parameters `--` but the online reference has it as single `-`, which is right? Do both work?

**Comment by bqminh** [comment]
I just checked and "blamed" the code:

https://github.com/iqtree/iqtree2/blame/c8be3c3117a0511a180c83a0d1768c0b3b520f41/utils/tools.cpp#L4914

This shows that I changed this option to `--sprrad` and `--radius` (they are synonymous) with the commit 77729c5 from Oct 29, 2018. Typically I keep old option untouched for backward compatibility and only add new option. I can't remember why I mistakenly changed `-sprrad` to `--sprrad` (sorry about that). But I believe all other options are untouched (hopefully!)

For the 2nd option:

https://github.com/iqtree/iqtree2/blame/c8be3c3117a0511a180c83a0d1768c0b3b520f41/utils/tools.cpp#L2349

`-czb` is the same as `--polytomy`. So both are correct (and doing the same thing).
